utils/firestore.js

```diff
@@ -109,6 +109,16 @@ const updateResponse = async (data, uid) => {
         const response = await db.collection('participants').where('state.uid', '==', uid).get();
         if(response.size === 1) {
             for(let doc of response.docs){
+                const previousQuery = doc.data().query;
+                const firstName = data[399159511]?.trim()?.toLowerCase();
+                const lastName = data[996038075]?.trim()?.toLowerCase();
```

Comment:
see other comment and please move outside of this function

Reply:
moved to submission.js